docs/CONTRIBUTING.md

```diff
@@ -55,6 +58,7 @@ uv venv
 source .venv/bin/activate
 uv sync          # Installs package in editable mode
 uv sync --dev    # Include dev dependencies if defined
+uv sync --extra lint # Include linting dependencies (ruff, yamllint)
```

Comment:
Optional: `--dev` should likely include the lint ones too.

Reply:
I have updated to remove the uv sync --extra lint , since the dev depedencies already includes lint.